.github/workflows/gosec.yml

```diff
@@ -13,12 +13,12 @@ jobs:
       GO111MODULE: on
     steps:
       - name: Checkout Source
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Rename this PR to Update gosec actions  or maybe just put this into https://github.com/grddev/aeron-go/pull/1 and rename that one to mention it updates gh actions / workflows, too?

In that other PR I already think the other two files touched in this PR should be moved into it.